addons/account/tests/test_tax.py

```diff
@@ -72,102 +72,16 @@ def test_tax_repartition_lines_intracomm_tax(self):
                     self._check_compute_all_results(
                         tax,
                         {
-                            'total_included': sign * 121.0,
-                            'total_excluded': sign * 100.0,
+                            'total_included': sign * price_unit,
+                            'total_excluded': sign * price_unit,
                             'taxes': (
-                                (sign * 100.0, sign * 21.0),
-                                (sign * 100.0, -sign * 21.0),
+                                (sign * price_unit, sign * 21.0),
+                                (sign * price_unit, -sign * 21.0),
                             ),
                         },
                         sign * price_unit,
                     )
 
-    def test_tax_repartition_lines_dispatch_amount_1(self):
```

Comment:
No for 2 reasons:
- We no longer can create a tax having 350% because we no longer scale the total amount of the tax with the total percentage of repartition lines.
- Making the repartition lines more important than the tax itself was one of the big reason why the round globally wasn't working well before.

Short answer: Now the total factor is always 100%, It's easier to manage the rounding issues and I no longer need to manage the granularity level of repartition lines in taxes aggregator.

The equivalent of this test would be to test a tax +50+50 and see if an odd number is well spread onto those tax lines. Maybe I could add a test like that...

Reply:
yes, that would be nice

Reply:
if you don't have time, after the OXP. Create a task or a #TODO